projects/angular/src/data/datagrid/datagrid-row.html

```diff
@@ -132,11 +132,13 @@
             type="button"
             #detailButton
             class="datagrid-detail-caret-button"
+            aria-haspopup="dialog"
+            [disabled]="detailDisabled"
+            *ngIf="!detailHidden"
             [class.is-open]="detailService.isRowOpen(item)"
             [attr.aria-label]="detailService.isRowOpen(item) ? clrDgDetailCloseLabel : clrDgDetailOpenLabel"
             [attr.aria-expanded]="detailService.isRowOpen(item)"
             [attr.aria-controls]="detailService.id"
-            aria-haspopup="dialog"
```

Comment:
I think this should be moved back. The aria attributes were groped together.

Reply:
Corrected in commit cea931f